vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -19491,6 +19583,15 @@ unit:V-SEC-PER-M
   rdfs:label "V SEC PER M" ;
   skos:prefLabel "volt second per metre" ;
 .
+unit:V2-HR
```

Comment:
As with Ampere-squared-hour, I recommend we put in a holding pen and seek out an expert.